File: aspnetcore-entityframework/aspnetcore-entityframework/Models/Employee.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace aspnetcore_entityframework.Models;

public class Employee
{
    public Guid Id { get; set; }
    public required string Name { get; set; }
    public required string Email { get; set; }
    public string? Phone { get; set; }

    [Column(TypeName = "decimal(18,2)")]
    public decimal Salary { get; set; }
}

File: java-spring-postgresql/src/main/java/com/spring/entity/Employee.java
package com.spring.entity;

import java.util.UUID;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "Employees")
public class Employee {
    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    private UUID id;

    @Column(name = "Name")
    private String name;

    @Column(name = "Email")
    private String email;

    @Column(name = "Phone", nullable = true)
    private String phone;

    @Column(name = "Salary")
    private float salary;
}
